### ISSUE TITLE
NTR: Peptidyl-arginine methylation, to form monomethylarginine

### ISSUE THREAD

**Issue by hbye** [geneontology/go-ontology#13012]
Hello,

Please may I request the following new term:

"Peptidyl-arginine methylation, to form monomethylarginine."
Alternatively, this could be "Peptidyl-arginine methylation, to form methylarginine" (as is phrased in GO:0042037) or perhaps "Peptidyl-arginine methylation, to monomethylarginine" as is phrased in GO:0019918 and  GO:0019919. 

This relates to the PMID:21531333, whereby methyltransferases prmt-1 and prmt-5  have been shown to catalyse the formation of monomethylarginine intermediates (as well as asymmetrical-dimethyl arginine and symmetrical-dimethyl arginine intermerdiates, respectively, for which terms already exist). 

I think it would be another child of "GO:0035247 peptidyl-arginine omega-N-methylation". A possible definition could be "The process of methylation of peptidyl-arginine to form peptidyl-N(omega)-monomethyl-L-arginine."

Thank you very much.

Best wishes,

Hema

**Comment by ukemi** [comment]
I think this is covered by the molecular function since it is a single gene product working on a single molecule. It makes sense that the other terms to which you refer are processes because there are two methylation steps in the formation of the product.

**Comment by hbye** [comment]
Hello,

Thank you for your feedback. 

I think a corresponding PROCESS would also be helpful for 
FUNCTION GO:0035241 protein-arginine omega-N monomethyltransferase activity
as the production of monomethylarginine, although described as an intermediate, is sometimes also the end product (fig 1 of PMID:21531333).

Also, I noticed that the following PROCESS terms exist for FUNCTION terms for asymmetric and symmetric di-methyl-arginine. In these cases, is not the PROCESS implied in the FUNCTION terms?

FUNCTION GO:0035242 protein-arginine omega-N asymmetric methyltransferase activity
PROCESS  GO:0019919 peptidyl-arginine methylation, to asymmetrical-dimethyl arginine

FUNCTION GO:0035243 protein-arginine omega-N symmetric methyltransferase activity
PROCESS GO:0019918 peptidyl-arginine methylation, to symmetrical-dimethyl arginine

Best wishes,

Hema

**Comment by ukemi** [comment]
Hi Hema,

In the latter two cases that you point out, the process includes the addition of both methyl groups to the arginine, so it is the result of the execution of two molecular function events. In the case of the term you are requesting, the formation of the monomethylated protein is the result of a single execution of a molecular function, whether it is an intermediate or a final product. So it seems to me that the function is the same as the process. There is an effort to move these types of processes out of GO #12859. We are going to discuss this approach at the next consortium meeting, but in this case I strongly feel that what is being requested here is simply a function. I am including @drabkin and @deustp01 in the discussion becasue I would value their comments as well.

**Comment by deustp01** [comment]
Nothing useful to add from here about the boundary between function and process.

**Comment by hbye** [comment]
Hello,

Thank you for your responses. I understand your points. However, I was thinking about this from an end users' point of view. I thought it might be useful to know the product of single execution- in this case it would be "monomethylarginine" which can also be an intermediate for the next reaction. Yes, indeed, if terms similar to the one I have requested are also to be moved out, then I understand that there is no need to add the new term I requested :)

Best wishes,

Hema
